Build /managed/firefighting-support page

petesfrench opened the pull request:
## Done

[Design](https://www.figma.com/file/elTbjLdEYglzrRX01EvGd2/24.04-U.com---Firefighting-Support?type=design&node-id=40-1615&mode=design&t=iuq6Qzen1bUXU2Fd-0)
[Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)

## QA

- Go to https://ubuntu-com-13845.demos.haus/managed/firefighting-support
- Compare content against [Copydoc](https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit)
- Check on different screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-9794


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

webteam-app commented:
### [<img src='https://assets.ubuntu.com/v1/6baef514-ubuntu-circle-of-friends-large.svg' height=32 width=32> Demo</img>](https://ubuntu-com-13845.demos.haus)
### [<img src='https://assets.ubuntu.com/v1/e512b0e2-jenkins.svg' height=32 width=32> Jenkins </img>](https://jenkins.canonical.com/webteam/job/start-demo/20121)
### [<img src='https://assets.ubuntu.com/v1/7144ec6d-logo-jaas-icon.svg' height=32 width=32> demos.haus </img>](https://demos.haus)

carkod commented:
@camillet16 can you review first? So that I can code review with the final settled code.

carkod commented on templates/managed/firefighting-support.html:
```diff
@@ -0,0 +1,535 @@
+    </div>
+    <div class="p-section">
+      <div class="u-fixed-width u-align--center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/81320e0d-smart-city.png",
+          alt="",
+          width="916",
+          height="526",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
```
Are we not hiding such illustrations for smaller screens? They are pretty big and increase scrolling uncessesarily

petesfrench replied:
@camillet16

petesfrench commented:
Hey @camillet16, I have addressed your comments. The only thing I did change is the text change, as that is how it appears in the doc and we should keep it as the source of truth. If we want to apply this text change we should go through the pdm first.

carkod commented:
@petesfrench djlint is not passing because of an unclosed `hr`
